### ISSUE TITLE
Evaluated Errors: SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by YHDang** [codalab/codalab-competitions#3259]
Hello, I met the problem as follows when I submit the testing results about COCO keypoint detection.
![image](https://user-images.githubusercontent.com/34653678/192207084-ff9c2bb6-e718-46b9-9bcf-079341d20254.png)

Is this a problem with the system or is there an error in the submitted result file?

**Comment by dtuantran** [comment]
Hello @YHDang ,
In my opinion, the error comes from your `run.py` file at line 33. Please verify your python syntax.
Regards,

**Comment by mitchelldehaven** [comment]
I don’t think this should be closed. They just upload their predictions, not the run.py file.

**Comment by YHDang** [comment]
> I don’t think this should be closed. They just upload their predictions, not the run.py file.

Yeah, you're right. I use the same code to generate the testing json file. I successfully submitted the file the first two times, but have failed after that.

**Comment by mitchelldehaven** [comment]
I think this is a site problem. I had the same issue in another competition (I even used an old file that was previously accepted and this had an error).

**Comment by YHDang** [comment]
> I think this is a site problem. I had the same issue in another competition (I even used an old file that was previously accepted and this had an error).

Yep, I think so. I just also submitted an old file (which was accepted by the old competition Web), but I also encountered this problem.

**Comment by Didayolo** [comment]
Here is the issue and fix for this:

- #3263
